examples/widgets/obj/lv_example_obj_2.c

```diff
@@ -6,6 +6,10 @@ static void drag_event_handler(lv_event_t * e)
     lv_obj_t * obj = lv_event_get_target(e);
 
     lv_indev_t * indev = lv_indev_get_act();
+    if(indev == NULL) {
```

Comment:
On other places the one-line format is used:
```c
    if(indev == NULL)  return;
```

For consistency please use it here too.

Reply:
done